frontend/src/composables/useEntriesAPI.ts

```diff
@@ -0,0 +1,29 @@
+import type { Entry } from '@/types/entry'
+
+export function useEntriesAPI() {
+  class EntryAddFailure extends Error {
+    constructor() {
+      super()
+    }
+  }
+
+  const useAddEntryAPI = async (entry: Entry) => {
+    console.log(JSON.stringify(entry))
+    const response = await fetch('http://localhost:3000/entries', {
```

Comment:
本当はProxyを使って、
```suggestion
    const response = await fetch('/entries', {
```
で書きたいですね　多重管理になるので

Reply:
@kentosity 

下記で修正しました。
[8a88c0e](https://github.com/wannabehime/training_simplified_wecall/pull/16/commits/8a88c0e3972e797305ec98e547aa0c89b7d36231)